fashrevwall/TwitterClient.py

```diff
@@ -0,0 +1,62 @@
+"""
+Represents a client that connects to Twitter to retrieve tweets based on a
+search criteria using Twitter REST API and Tweepy.
+"""
+import json
+import tweepy
+from tweepy import OAuthHandler
+from TwitterStreamListener import TwitterStreamListener
+
+class TwitterClient:
+    def __init__(self):
+        self.api = self._get_twitter_api()
+
+    def _get_twitter_api(self):
+        """
+        Since we are only reading public information from Twitter, we don't need
+        access token/secret values.
+        """
+        with open('secrets.json') as secrets_file:
```

Comment:
can we change these to be picked up for env variables? then we can also easily set those up in heroku